backend/django/core/utils/utils_external_db.py

```diff
@@ -25,6 +26,24 @@ def get_connection(db_type, connection_dict):
         raise ValueError(f"ERROR: database type {db_type} is invalid")
 
 
+def test_login(db_type, engine_database):
+    """Attempts to login with given connection."""
+    try:
+        # Perform a no-op
+        pd.read_sql("NULL", con=engine_database)
```

Comment:
Getting this error when I put in the correct password
<img width="1621" alt="Screen Shot 2022-07-18 at 3 48 57 PM" src="https://user-images.githubusercontent.com/10951569/179605272-e5e5ab3b-b7c8-4dce-aeed-57d7b55c00bb.png">